themes/default/content/docs/get-started/aws/deploy-changes.md

```diff
@@ -117,10 +115,426 @@ $ aws s3 ls $(pulumi stack output bucketName)
+
+```bash
+2020-08-27 12:30:24         70 index.html
+```
+
+Now that `index.html` is in the bucket, modify the program to serve `index.html` as the home page of the website.
+
+{{< chooser language "javascript,typescript,python,go,csharp,java,yaml" / >}}
+
+{{% choosable language "javascript,typescript" %}}
+
+```typescript
+const bucket = new aws.s3.Bucket("my-bucket", {
+    website: {
+        indexDocument: "index.html",
+    },
+});
+```
+
+{{% /choosable %}}
+
+{{% choosable language python %}}
+
+```python
+bucket = s3.Bucket("my-bucket",
+    website=s3.BucketWebsiteArgs(
+        index_document="index.html",
+    ),
+)
+```
+
+{{% /choosable %}}
+
+{{% choosable language go %}}
+
+```go
+bucket, err := s3.NewBucket(ctx, "my-bucket", &s3.BucketArgs{
+    Website: &s3.BucketWebsiteArgs{
+        IndexDocument: pulumi.String("index.html"),
+    },
+})
+if err != nil {
+    return err
+}
+```
+
+{{% /choosable %}}
+
+{{% choosable language csharp %}}
+
+```csharp
+var bucket = new Aws.S3.Bucket("my-bucket", new()
+{
+    Website = new Aws.S3.Inputs.BucketWebsiteArgs
+    {
+        IndexDocument = "index.html",
+    },
+});
+```
+
+{{% /choosable %}}
+
+{{% choosable language java %}}
+
+```java
+import com.pulumi.aws.s3.BucketArgs;
+import com.pulumi.aws.s3.inputs.BucketWebsiteArgs;
+
+var bucket = new Bucket("bucket", BucketArgs.builder()
+    .website(BucketWebsiteArgs.builder()
+        .indexDocument("index.html")
+        .build())
+    .build());
+```
+
+{{% /choosable %}}
+
+{{% choosable language yaml %}}
+
+```yaml
+my-bucket:
+  type: aws:s3:Bucket
+  properties:
+    website:
+      indexDocument: index.html
+```
+
+{{% /choosable %}}
+
+Finally, you'll make a few adjustments to allow these resources to be accessed anonymously over the Internet.
+
+For the bucket itself, you'll need two new resources: a `BucketOwnershipControls` resource, which defines the bucket's object-ownership settings, and a `BucketPublicAccessBlock` resource, which allows the bucket to be accessed publicly.
+
+For the `BucketObject`, you'll need an access-control (ACL) setting of `public-read` to allow the page to be accessed publicly, and a content type of `text/html` to tell AWS to serve the file as a web page:
+
+{{< chooser language "javascript,typescript,python,go,csharp,java,yaml" / >}}
+
+{{% choosable language "javascript,typescript" %}}
+
+```typescript
+const ownershipControls = new aws.s3.BucketOwnershipControls("ownership-controls", {
+    bucket: bucket.id,
+    rule: {
+        objectOwnership: "ObjectWriter"
+    }
+});
+
+const publicAccessBlock = new aws.s3.BucketPublicAccessBlock("public-access-block", {
+    bucket: bucket.id,
+    blockPublicAcls: false,
+});
+
+const bucketObject = new aws.s3.BucketObject("index.html", {
+    bucket: bucket.id,
+    source: new pulumi.asset.FileAsset("index.html"),
+    contentType: "text/html",
+    acl: "public-read",
+}, { dependsOn: ownershipControls });
```

Comment:
All the other code examples specify the public access block for `DependsOn`, should this one match the others?

Reply:
Yes -- good catch!